jaxrs-clients/src/test/java/com/palantir/remoting2/jaxrs/TestBase.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting2.jaxrs;
+
+import com.google.common.collect.ImmutableList;
+import com.palantir.remoting.api.config.ssl.SslConfiguration;
+import com.palantir.remoting2.clients.ClientConfiguration;
+import com.palantir.remoting2.clients.ClientConfigurations;
+import com.palantir.remoting2.config.ssl.SslSocketFactories;
+import java.nio.file.Paths;
+
+public abstract class TestBase {
+
+    protected final ClientConfiguration createTestConfig(String... uri) {
```

Comment:
slight preference for making this a static utils class (`TestClientConfigurations.createTestConfig`) since it's not stateful